README.md

```diff
@@ -133,6 +133,30 @@ pkg> build Conda
 Also note that you have to use Miniforge for `aarch64-linux-gnu` and
 `aarch64-apple-darwin` platforms as Miniconda is not available for those platforms yet.
 
+
+## Troubleshooting
+
+### Installation on Windows with special characters in user names
+
+If you have a special character in your user name (like an umlaut, an accent or a space) the installation which defaults to
+directory `C:\Users\<username>\.julia\Conda\3` will fail on Windows.
+This is a [known issue](https://github.com/conda/conda/issues/10239). The work-around is to install Miniconda to a user-writable directory outside of the home directory.
+If you have an unsuccessful installation of `Conda.jl` and/or related packages like `IJulia` or `PyPlot`, it might be necessary to remove these packages first.
+Before installing `Conda.jl`, choose a directory without space and without special characters and set the environment variable `CONDA_JL_HOME` as follows inside a julia session:
+
+```julia
+ENV["CONDA_JL_HOME"] = "C:\\Conda-julia\\3"
+
+# Install Conda.jl
+using Pkg
+Pkg.add("Conda")
```

Comment:
If they have already added Conda (which seems likely if they are troubleshooting), they should use `Pkg.build` here.